app/src/main/kotlin/com/wire/android/util/CurrentScreenManager.kt

```diff
@@ -45,18 +54,27 @@ class CurrentScreenManager @Inject constructor() : DefaultLifecycleObserver, Nav
 
     override fun onResume(owner: LifecycleOwner) {
         super.onResume(owner)
-        isAppVisibleFlow.value = true
+        appLogger.i("${TAG}: onResume called")
+        isAppVisibleFlow.value = true && isScreenUnLocked
```

Comment:
I think this conflicts with #1007. Where the `isAppVisibleFlow` is used to start/stop sync. It shouldn't really matter if the screen is locked or not, I'm not sure.

Perhaps we could just rename `isAppVisibleFlow` to `isOnForegroundFlow`.
And on `fromNavigationItem`, we just add an extra check to `isScreenUnlocked`